src/page_body/structure/blocks/page_block_table.pr

```diff
@@ -10,7 +10,7 @@
 {[ let tableWidthStyle = "" /]}
 
 {* This is hardcoded value on backend when the user sets "Fit to page width" *}
-{[ if (tableWidth === 751 || tableWidth === 752) ]}
+{[ if (tableWidth === 751 || tableWidth === 752 || tableWidth === 753) ]}
```

Comment:
Why don't we just calculate this based on number of table columns?

Reply:
Yeah, good call. I create [another ticket](https://linear.app/supernova-io/issue/PF-48/ex-docs-or-calculate-full-width-tables-properly) for it to solve this tech debt